src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/DefaultDocumentVersionCache.cs

```diff
@@ -102,6 +103,23 @@ public override void Initialize(ProjectSnapshotManagerBase projectManager)
         {
             _projectSnapshotManager = projectManager;
             _projectSnapshotManager.Changed += ProjectSnapshotManager_Changed;
+
+        }
+
+        public override void RazorFileChanged(string filePath, RazorFileChangeKind kind)
+        {
+            _foregroundDispatcher.AssertForegroundThread();
+
+            switch (kind)
+            {
+                case RazorFileChangeKind.Removed:
+                    if (_documentLookup.ContainsKey(filePath))
```

Comment:
I don't believe this `filePath` is run through our `FilePathNormalizer.Normalize`. I imagine this would break in windows cases because `filePath` would have `\` and the document lookup table would have `/` right?